cypress_shared/pages/apply/attachDocumentsPage.ts

```diff
@@ -1,5 +1,4 @@
 import { ApprovedPremisesApplication, Document } from '@approved-premises/api'
```

Comment:
I ran into the same issue, but solved it with some changes in the cypress config - https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/pull/74/commits/5632be8f91d7feba056e5a0a9c8418ccc8b4894c